test/oli/groups_test.exs

```diff
@@ -14,12 +14,12 @@ defmodule Oli.GroupsTest do
           name: "Testing name",
           description: "Testing description",
           key_contact: "Testing key contact",
-          prohibit_global_access: false})
+          global_access: false})
 
       assert community.name == "Testing name"
       assert community.description == "Testing description"
       assert community.key_contact == "Testing key contact"
-      assert community.prohibit_global_access == false
+      assert community.global_access == false
```

Comment:
I would like to see use of the factory here to remove hard-coded assertions such as:

`
params = params_for(:community)
assert {:ok, %Community{} = community} = Groups.create_community(params)

assert community.name == params.name
etc
etc
`